readme.md

```diff
@@ -21,13 +21,33 @@ compileWithBabel: [
 ```
 
 ## Usage
+### Data requirements
+The components are meant to be used with the Sanity `image` type, but you do have to dereference the asset field. The asset fields contains information that is used to determine the correct image sizes to fetch and to add the correct width and height attributes to the `img` tag on the server, thus preventing [CLS](https://web.dev/cls/).
```

Comment:
Ola, we weten op de server toch niet op welk formaat het wordt weergegeven? Waarschijnlijk mis ik iets, maar hoe voorkomt dit CLS?

Reply:
We weten inderdaad niet het weergaveformaat, maar we kunnen wel de afbeelding van het juiste `width` en `height` attribuut voorzien. Hiermee kan de browser (samen met de CSS) direct bepalen hoeveel ruimte het plaatje inneemt, zonder dat het hiervoor geladen hoeft te worden. 

Bij `ImageCover` werkt dit overigens niet, omdat je daar niet weet wat `width` en `height` worden, omdat dat niet afhankelijk is van de dimensions van de afbeelding.